packages/renderer/src/lib/image/ImageDetailsFiles.svelte

```diff
@@ -11,16 +11,17 @@ import FilesystemLayerView from './FilesystemLayerView.svelte';
 import { type ImageFilesystemLayerUI, toImageFilesystemLayerUIs } from './imageDetailsFiles';
 import ImageDetailsFilesLayers from './ImageDetailsFilesLayers.svelte';
 
-export let imageInfo: ImageInfo | undefined;
+let { imageInfo }: { imageInfo: ImageInfo | undefined } = $props();
+
+let imageLayers: ImageFilesystemLayers | undefined = $state();
+let selectedLayer: ImageFilesystemLayerUI | undefined = $state();
```

Comment:
```suggestion
let imageLayers: ImageFilesystemLayers | undefined = $state(undefined);
let selectedLayer: ImageFilesystemLayerUI | undefined = $state(undefined);
```

Are we sure svelte is using `undefined` and not null?